sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -399,60 +399,60 @@ def __init__(self, *, key: str, label: Optional[str] = None) -> None:
         self.label = label
 
 
-class Snapshot:  # pylint: disable=too-many-instance-attributes
+class ConfigurationSnapshot:  # pylint: disable=too-many-instance-attributes
     """A point-in-time snapshot of configuration settings."""
 
     name: Optional[str]
-    """The name of the snapshot."""
+    """The name of the configuration snapshot."""
     status: Optional[Union[str, SnapshotStatus]]
     """The current status of the snapshot. Known values are: "provisioning", "ready",
         "archived", and "failed"."""
-    filters: List[ConfigurationSettingFilter]
-    """A list of filters used to filter the key-values included in the snapshot. Required."""
+    filters: List[SnapshotSettingFilter]
+    """A list of filters used to filter the key-values included in the configuration snapshot."""
     composition_type: Optional[str]
-    """The composition type describes how the key-values within the snapshot
+    """The composition type describes how the key-values within the configuration snapshot
         are composed. The 'key' composition type ensures there are no two key-values containing the
         same key. The 'key_label' composition type ensures there are no two key-values containing the
         same key and label. Known values are: "key" and "key_label"."""
     created: Optional[datetime]
-    """The time that the snapshot was created."""
+    """The time that the configuration snapshot was created."""
     expires: Optional[datetime]
-    """The time that the snapshot will expire."""
+    """The time that the configuration snapshot will expire."""
     retention_period: Optional[int]
-    """The amount of time, in seconds, that a snapshot will remain in the
-        archived state before expiring. This property is only writable during the creation of a
+    """The amount of time, in seconds, that a configuration snapshot will remain in the
+        archived state before expiring. This property is only writable during the creation of a configuration
         snapshot. If not specified, the default lifetime of key-value revisions will be used."""
     size: Optional[int]
-    """The size in bytes of the snapshot."""
-    item_count: Optional[int]
-    """The amount of key-values in the snapshot."""
+    """The size in bytes of the configuration snapshot."""
+    items_count: Optional[int]
```

Comment:
item_count?

Reply:
We don't want to change the name in defined in swagger.

Reply:
But we also renamed the `Snapshot` to `ConfigurationSnapshot` which `Snapshot` is the named fn defined in Swagger as well.

Reply:
Because that's the class name. We would keep the property names defined in swagger.